docs/deployments/azure/running-azure-powershell/index.md

```diff
@@ -36,6 +38,13 @@ Azure supports two authentication methods, each of which provides access to a di
 
 Learn more about [configuring the right Azure Account](/docs/infrastructure/deployment-targets/azure/index.md).
 
+## Dynamic Worker Pools
+
+Octopus Cloud uses a special type of worker pool called a [Dynamic Worker Pool](/docs/infrastructure/workers/dynamic-worker-pools.md). These are provided by Octopus and custom versions of the Azure tools cannot be installed on them. In order to use your own version of the Azure CLI or Azure Powershell cmdlets when using Dynamic Worker Pools, please do the following:
```

Comment:
`cannot be installed` -> `cannot be easily installed`.  Technically speaking someone could add a piece of code to every step of their process that installs Azure dependencies if they are not installed.